package.json

```diff
@@ -21,8 +21,12 @@
       "default": "./lib/browser/index.js"
     },
     "./node": {
-      "browser": null,
       "types": "./lib/node/index.d.ts",
+      "node": {
```

Comment:
Also, a reverse fix for `/browser`? Adding a `browser` field with a higher priority. Folks experience the reverse problem as well for `msw/browser`.